tests/ctypesgentest.py

```diff
@@ -220,17 +221,19 @@ def _compile_common(common_lib):
     )
 
 
-def _generate_common(file_name, common_lib, embed_preamble=True):
+def _generate_common(file_name, common_lib, embed_preamble=True, link=False):
```

Comment:
note: also much nicer in the new codebase, where we just let ctypesgentest call into the `main()` API directly.
See https://github.com/pypdfium2-team/ctypesgen/blob/pypdfium2/tests/conftest.py